ledfx/virtuals.py

```
@@ -317,6 +317,7 @@ def update_segments(self, segments_config):
             _LOGGER.debug(
                 f"Virtual {self.id}: updated with {len(self._segments)} segments, totalling {self.pixel_count} pixels"
             )
+            self._ledfx.virtuals.check_and_deactivate_devices
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)